build_calicoctl/requirements.txt

```diff
@@ -14,5 +14,5 @@ netaddr==0.7.15
 flask
 gunicorn
 subprocess32
-git+https://github.com/projectcalico/python-etcd.git
+git+https://github.com/jplana/python-etcd.git
```

Comment:
Actually, I don't think we need this here either.  `calicoctl` shouldn't be directly importing it, so just let the `libcalico` dep sort it out.